net-im/corebird/corebird-1.7.4.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+VALA_MIN_API_VERSION=0.34
+
+inherit autotools gnome2 vala virtualx xdg-utils
+
+DESCRIPTION="Native GTK+3 Twitter client"
+HOMEPAGE="https://corebird.baedert.org/"
+SRC_URI="https://github.com/baedert/corebird/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug gstreamer spellcheck"
+
+RDEPEND="dev-db/sqlite:3
+	>=dev-libs/glib-2.44:2
+	dev-libs/json-glib
+	gstreamer? ( >=media-libs/gst-plugins-bad-1.6:1.0[X,gtk]
+		media-libs/gst-plugins-base:1.0[X]
+		media-libs/gst-plugins-good:1.0
+		media-plugins/gst-plugins-hls:1.0
+		media-plugins/gst-plugins-libav:1.0
+		media-plugins/gst-plugins-meta:1.0[X]
+		media-plugins/gst-plugins-soup:1.0 )
+	spellcheck? ( >=app-text/gspell-1.0[vala] )
+	>=net-libs/libsoup-2.42.3.1
```

Comment:
This version no longer exists as well and the latest version available is at least 2.56.1. Please update this line to `net-libs/libsoup` then.